<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertingSettings</class>
 <widget class="QDialog" name="InsertingSettings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="types">
       <property name="title">
        <string>Тип</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="isstring">
          <property name="text">
           <string>Строка настроек</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isswitcher">
          <property name="text">
           <string>Переключатель</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isbutton">
          <property name="text">
           <string>Кнопка</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="MyLineEdit" name="abbr">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Аббревиатура</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MyLineEdit" name="param">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="MyLineEdit" name="metric">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Единицы измерения</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="abbr_colour">
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зеленый</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="value_colour">
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зелёный</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="metric_colour">
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зелёный</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="MyLineEdit" name="abbr_2">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Аббревиатура</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="MyLineEdit" name="param_2">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="MyLineEdit" name="metric_2">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Единицы измерения</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="abbr_2_colour">
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зелёный</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="value_2_colour">
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зелёный</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="metric_2_colour">
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зелёный</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="MyLineEdit" name="param_header_name">
       <property name="placeholderText">
        <string>Название параметра кадра</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="MyLineEdit" name="switcher_name">
         <property name="font">
          <font>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Введите название переключателя</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MyLineEdit" name="switcher">
         <property name="font">
          <font>
           <italic>false</italic>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>перечислите через &quot;/&quot; варианты переключений (например, ааа/БББ/ввв)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="MyLineEdit" name="name">
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>false</underline>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Введите название кнопки</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="MyLineEdit" name="header_name">
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>Введите название кадра</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="specialsymbols">
       <property name="title">
        <string>Специальные символы</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="MyButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Α</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="MyButton" name="pushButton_2">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Β</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="MyButton" name="pushButton_3">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ɣ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="MyButton" name="pushButton_4">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Δ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="MyButton" name="pushButton_5">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Λ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="MyButton" name="pushButton_6">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Θ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="MyButton" name="pushButton_7">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Μ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="MyButton" name="pushButton_8">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Φ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="MyButton" name="pushButton_9">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ψ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="MyButton" name="pushButton_10">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ω</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MyButton" name="pushButton_11">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>⪻</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MyButton" name="pushButton_18">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>⪼</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="MyButton" name="pushButton_16">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string>Нижний индекс двойки</string>
          </property>
          <property name="text">
           <string>₂</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="MyButton" name="pushButton_14">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string>Степень тройки</string>
          </property>
          <property name="text">
           <string>³</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="MyButton" name="pushButton_19">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string>Степень двойки</string>
          </property>
          <property name="text">
           <string>²</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="MyButton" name="pushButton_12">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="whatsThis">
           <string>Нижняя индекс тройки</string>
          </property>
          <property name="text">
           <string>₃</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="MyButton" name="pushButton_20">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>°</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="MyButton" name="pushButton_13">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>±</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="MyButton" name="pushButton_17">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>α</string>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="MyButton" name="pushButton_15">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>β</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="MyButton" name="pushButton_21">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>γ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="MyButton" name="pushButton_22">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>δ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="MyButton" name="pushButton_23">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>λ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="MyButton" name="pushButton_24">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>μ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="MyButton" name="pushButton_25">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>θ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="MyButton" name="pushButton_26">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>φ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="MyButton" name="pushButton_27">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ψ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="MyButton" name="pushButton_28">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ω</string>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="MyButton" name="pushButton_29">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ρ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="MyButton" name="pushButton_30">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ρ</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header location="global">button.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
